### PR TITLE
fix machine naming conventions

### PR DESCRIPTION
**Pull request by baude**
try to align the machine commands and their usage descriptions.

[NO TESTS NEEDED]

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: cmd/podman/machine/list.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ var (
 		Use:     "list [options]",
 		Aliases: []string{"ls"},
 		Short:   "List machines",
-		Long:    "List Podman managed virtual machines.",
+		Long:    "List machines.",
```
Comment (TomSweeneyRedHat) [inline cmd/podman/machine/list.go:31]: @rhatdan @baude, I don't understand why you're reverting this.

Comment (baude) [reply cmd/podman/machine/list.go:31]: i believe @rhatdan preferred the more direct use

Comment (rhatdan) [reply cmd/podman/machine/list.go:31]: I am fine with the long discription. as managed virtual machines.  Better to have the long description to have more data then the short one.

Comment (rhatdan) [reply cmd/podman/machine/list.go:31]: Although I agree with removing Podman.

Comment (rhatdan) [reply cmd/podman/machine/list.go:31]: There should be consistency though, so above should be "Initialize a virtual machine"

**Comment by rhatdan** [comment]
LGTM

**Comment by baude** [comment]
merge me

**Pull request merged**